db/db_filesnapshot.cc

```diff
@@ -372,8 +372,13 @@ Status DBImpl::GetLiveFilesStorageInfo(
 
   size_t wal_size = live_wal_files.size();
 
+  std::ostringstream wal_names;
```

Comment:
do you think we may want to do this only in the DEBUG build?

Reply:
Sure fixed.